src/tasks/task-module.js

```diff
@@ -218,7 +220,7 @@ class TaskModule extends BaseModule {
       for (let taskIndex = 0; taskIndex < workers; taskIndex++) {
         let taskInfo = infoForTasks[taskIndex];
         if (!taskInfo)
-          taskInfo = infoForTasks[taskIndex] = { failedCount: 0, promise: null, stop: false };
+          taskInfo = infoForTasks[taskIndex] = { failedCount: 0, promise: null, stop: false, nextRunAt: taskKlass.nextRun(taskIndex, undefined, Date.now()) };
```

Comment:
Let's stick with either `Date` or `DateTime`, and not mix them. Why? Because it is possible that different timezone settings will be applied globally for `DateTime`.